wdl2cwl/main.py

```diff
@@ -1071,8 +1061,9 @@ def get_cwl_task_outputs(
                             glob=glob_str,
                             loadContents=True,
                             outputEval=r"""${
-if (self[0].contents == 'true') { return true;}
-if (self[0].contents == 'false') { return false;}
+var contents = self[0].contents.trim().toLowerCase()
+if (contents == 'true') { return true;}
+if (contents == 'false') { return false;}
```

Comment:
@kinow And indeed, the functional testing revealed issues with my implementation :-)